extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -403,6 +400,16 @@ export class AmpA4A extends AMP.BaseElement {
     return true;
   }
 
+  /** @return {!Array|!string} */
```

Comment:
Add some documentation indicating what these are for and when they should be overridden.  Also type should be `!Array<string>|string`